assets/src/edit-story/app/index.js

```diff
@@ -33,10 +34,12 @@ function App( { config } ) {
 					<APIProvider>
 						<HistoryProvider size={ 50 }>
 							<StoryProvider storyId={ storyId }>
-								<GlobalStyle />
-								<CropMoveableGlobalStyle />
-								<Layout />
-								<Popover.Slot />
+								<FontProvider>
+								  <GlobalStyle />
+								  <CropMoveableGlobalStyle />
+								  <Layout />
+								  <Popover.Slot />
+                </FontProvider>
```

Comment:
The indentation in this area is a bit off (spaces vs tabs)

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3998/commits/3ee619201154af1ef3c75c4dd1f55fcc1c3b2cde